runelite-client/src/main/java/net/runelite/client/plugins/dpscounter/DpsCounterPlugin.java

```diff
@@ -162,7 +162,12 @@ public void onHitsplatApplied(HitsplatApplied hitsplatApplied)
 		}
 
 		Hitsplat hitsplat = hitsplatApplied.getHitsplat();
-
+		final int npcId = ((NPC) actor).getId();
+		boolean isBoss = BOSSES.contains(npcId);
+		if (dpsConfig.bossOnly() && !isBoss)
+		{
+			return;
+		}
```

Comment:
Add some spacing between variable declarations and statement blocks. Also flip the conditions to avoid some config checks.
```suggestion
		final int npcId = ((NPC) actor).getId();
		boolean isBoss = BOSSES.contains(npcId);

		if (!isBoss && dpsConfig.bossOnly())
		{
			return;
		}

```